src/autopas/containers/TraversalInterface.h

```diff
@@ -7,13 +7,15 @@
 #pragma once
 
 #include "autopas/options/DataLayoutOption.h"
+#include "autopas/options/InteractionTypeOption.h"
 #include "autopas/options/TraversalOption.h"
 
 namespace autopas {
 
 /**
  * This interface serves as a common parent class for all traversals.
  */
+template <InteractionTypeOption::Value interactionType>
```

Comment:
@FG-TUM (ping for extra attention to this issue)
I tried it with getters, but the problem is that I need to differentiate between the two at compile time.
Because we also define our functors at compile time and they use different function declarations. e.g. AoSFunctor() taking 3 particles instead of 2.
Do you have a good idea how to handle this without a template?